hamilton/plugins/jupyter_magic.py

```diff
@@ -242,159 +409,13 @@ def insert_module(self, line):
 
         module_path = Path(line)
         # insert our custom %%with_functions magic at the top of the cell
-        header = f"%%cell_to_module -m {module_path.stem}\n\n"
+        header = f"%%cell_to_module {module_path.stem}\n"
         module_source = module_path.read_text()
-
         # insert source code as text in the next cell
         self.shell.set_next_input(header + module_source, replace=False)
 
-    @magic_arguments()  # needed on top to enable parsing
```

Comment:
We can't delete it -- we can chat about whether subsuming the functionality makes sense or not.